doc/USER_GUIDE.md

```
@@ -55,14 +55,24 @@ CloudAI is fully configurable via set of TOML configuration files. You can find
 Test definition is a Pydantic model that describes the arguments of a test. Such models should be inherited from the `TestDefinition` class:
 ```py
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)